integration/tests/healthcheck_test.go

```diff
@@ -0,0 +1,30 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHealthcheck(t *testing.T) {
+	tt := []struct {
+		Name string
+	}{
+		{
+			Name: "Health Check",
```

Comment:
Question: Do we see need to add more test cases here?

Reply:
Out of scope for my planned work. This is refactoring of what @akhalina did in here previous spike, unchanged.  The expectation is - there is a file per api endpoint, and this file is supposed to cover healthchecks across different schema versions.

Reply:
At least that's the pattern I followed so far

Reply:
Not really sure what we can add here. This was meant as more of a sample of what tests should look like.

Reply:
This one is useful as it is run for every "fixture" (i.e. full test suite run with different schema version). And if healthcheck fails, the rest of the tests are pretty much guaranteed to fail.

Reply:
I agree, definitely useful to get a general sense of if the service in the test is cooperating.

Reply:
Sounds good, since tests were parametrized that made me think we may want to add more test parameters in future